engine/sync_manager_types.go

```diff
@@ -47,8 +41,8 @@ type syncManager struct {
 	initSyncWG                     sync.WaitGroup
 	inService                      sync.WaitGroup
 
-	currencyPairs            map[currencyPairKey]*currencyPairSyncAgent
-	tickerBatchLastRequested map[string]time.Time
+	currencyPairs            map[key.ExchangePairAsset]*currencyPairSyncAgent
+	tickerBatchLastRequested map[key.ExchangeAsset]time.Time
```

Comment:
OUT OF SCOPE: We might need to upgrade `e.SupportsRESTTickerBatchUpdates()` to take in an asset type and then have a map of batching functionality to those types because from memory `Bitfinex?` has batching support for a couple assets and this might trip up the overall performance that this is trying to provide. Also this is awesome 🦖.

Reply:
The change above definitely isn't about performance, but to ensure each asset type gets called. [I noted in this comment](https://github.com/thrasher-corp/gocryptotrader/pull/1229#discussion_r1255022223) that we can skip data collection

I do agree though that such a breakdown would be good though